glui_translation.cpp

```diff
@@ -100,7 +100,16 @@ int    GLUI_Translation::iaction_mouse_down_handler( int local_x,
     }
     else {
       locked = GLUI_TRANSLATION_LOCK_NONE;
-      glutSetCursor( GLUT_CURSOR_SPRAY );
+	  	  
+	  //XC_spraycan (GLUT_CURSOR_SPRAY) is an air-brush icon. 
+	  //This crosshair has 8 directions, and is identical on
+	  //windows with freeglut. Don't know about classic GLUT.	  
+	  //THOUGHTS:
+	  //X11 has XC_fleur shaped like four-way arrows but not
+	  //strictly the same as the double-arrows.
+	  //Windows has the MOVE cursor, which would be an ideal.
+	  //Better the cursor could disappear so not to distract.
+      glutSetCursor(/*GLUT_CURSOR_SPRAY*/GLUT_CURSOR_FULL_CROSSHAIR);
```

Comment:
Do you suggest a "fullscreen" cross-hair?  Let's wait and see if there are reports. I think using GLUT for this is a bad idea. But a spray-can is just kind of absurd. If you want to change this I recommend using the NONE cursor so on some implementation the icon will be hidden and others it will just appear as the default cursor.

We can worry about it later. What's important is to not use a spray can and to not forget about it.